yahoo-japan/road-network.json

```diff
@@ -0,0 +1,151 @@
+        "description": "Major road networks and highway shields."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [141.5996707, 42.6487807]
+      },
+      "properties": {
+        "place_name": "Tomakomai, Japan",
+        "zoom": 18,
+        "description": "Traffic stops and oneway arrows."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [144.38677, 43.01335]
+      },
+      "properties": {
+        "place_name": "Kushiro, Japan",
+        "zoom": 15,
+        "description": "The city grid of Kushiro, Japan."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [135.76827, 35.49391]
+      },
+      "properties": {
+        "place_name": "Obama, Japan",
+        "zoom": 15,
+        "description": "Road exit signs."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [139.758, 35.675]
+      },
+      "properties": {
+        "place_name": "Tokyo, Japan",
+        "zoom": 12,
+        "description": "Tokyo's dense roads network."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [134.576208, 34.102612]
+      },
+      "properties": {
+        "place_name": "Tokushima, Japan",
+        "zoom": 16,
+        "description": "The intersection of motorways, trunks, and motorway links."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [133.568296, 33.557096]
+      },
+      "properties": {
+        "place_name": "Kochi, Japan",
+        "zoom": 18,
+        "description": "Bridges over water features."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [138.231664, 36.595319]
+      },
+      "properties": {
+        "place_name": "Nagano, Japan",
+        "zoom": 15,
+        "description": "Motorway tunnels."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [141.03488, 43.1791]
+      },
+      "properties": {
+        "place_name": "Otaru, Japan",
+        "zoom": 16,
+        "description": "Trunk tunnels."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [140.993642, 43.188037]
+      },
+      "properties": {
+        "place_name": "Otaru, Japan",
+        "zoom": 17,
+        "description": "Walking paths in a park."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [144.36847, 42.99832]
+      },
+      "properties": {
+        "place_name": "Kushiro, Japan",
+        "zoom": 12,
+        "description": "An city grid of Kushiro."
```

Comment:
Dupe of line `39`? At least, or if this is to evaluate at z15 and z12 maybe the coordinates should be the same?